Compilers/FreeBSD/gcc.ey

```diff
@@ -0,0 +1,31 @@
+%e-yaml
```

Comment:
@JoshDreamland at the time, LateralGM was crashing because it needed those gcc.ey's to exist in order to run. Creating the files allowed it to open without crashing. However, we've made changes since that happened. I'll see if the changes effect whether this is still needed.

Reply:
@JoshDreamland ok, i deleted  FreeBSD/gcc.ey and FreeBSD/gcc32.ey and no crash. Sounds like we must've fixed that since this was originally done.

Reply:
@JoshDreamland I stand corrected. I what i did differently last time i tried it, but i tried it again just now a clean install with no gcc.ey or gcc32.ey and it complained saying "Unsupported platform" but adding those two files back and running lgm again fixed that error. It seems lgm has it hardcoded the need for a gcc.ey file to both exist and be 100% readable. We've tried in the past renaming clang.ey to gcc.ey and that didn't work either.